VM/src/lbitlib.cpp

```diff
@@ -210,6 +212,16 @@ static int b_countrz(lua_State* L)
     return 1;
 }
 
+static int b_swap(lua_State* L)
+{
+    LUAU_ASSERT(FFlag::LuauBit32Byteswap);
```

Comment:
Similar here, but the difference is that it's harder to flag the addition of `b_swap` to `bitlib` array.

For a simpler solution, I would suggest
```
if (!FFlag::LuauBit32Byteswap)
    luaL_error(L, "bit32.byteswap isn't enabled");
```